### ISSUE TITLE
Create a 3.0.0-alpha.4 release

### ISSUE THREAD

**Issue by starksm64** [arquillian/arquillian-graphene#245]

**Comment by starksm64** [comment]
```bash
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.863 s
[INFO] Finished at: 2024-04-21T23:18:58-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.6.3:jar (attach-javadocs) on project graphene-webdriver-spi: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1
[ERROR] error: Class org.jboss.apiviz.APIviz is not a valid doclet.
[ERROR]   Note: As of JDK 13, the com.sun.javadoc API is no longer supported.
[ERROR] Command line was: /home/starksm/bin/jdk-17.0.11+9/bin/javadoc "-sourceclasspath /home/starksm/Dev/arquillian-graphene/spi/target/classes" @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/starksm/Dev/arquillian-graphene/spi/target/apidocs' dir.
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :graphene-webdriver-spi
starksm@fedora:~/Dev/arquillian-graphene1$ 

```

**Comment by lprimak** [comment]
I ran maven with `-Pjboss-release` with and without `-Darguments="-DskipTests"` with the above PR and it worked, including Javadoc

**Comment by starksm64** [comment]
It has been pushed to the jboss repo
https://github.com/arquillian/arquillian-graphene/releases/tag/3.0.0-alpha.4

**Comment by lprimak** [comment]
Hmmm... I don't see it in JBoss repo (yet?) Can you please double-check?

**Comment by starksm64** [comment]
Ok, the log says it published 3.0.0 snapshot versions. I guess I'll have to delete the tag and rerun the release.

**Comment by starksm64** [comment]
It is out there now:
https://repository.jboss.org/nexus/content/repositories/releases/org/jboss/arquillian/graphene/graphene-webdriver-impl/3.0.0-alpha.4/
